real_time/git_etiquette.md

```diff
@@ -1,20 +0,0 @@
-How we use git
-==============
-
-We use git as version control for all software written for robobuggy.
-
-For deployed code vs. undeployed code, we use git branches. 
-We expect that the code in the deployed branch is deployable at all times. 
-
-TODO stable, staging, dev branches?
-
-So you want to write some code...
-=================================
-TODO put workflow information here. 
-
-
-How to merge your code into mainline
-====================================
-We will use a git-rebase-style 
-
```

Comment:
I am moving this to a wiki page, I don't think it really belongs in the code.  It also should be repo wide not just real time